features/login.feature

```diff
@@ -55,10 +55,19 @@ Scenario: Ensure login shows proper error messages
 
     # try offline login with bad password
     Then I login with username "user_with_no_data" and password "bad pass", without waiting for completion
-    Then I see the text "Either the password you entered was incorrect, or CommCare couldn't reach the server"
+    # Check that you see "Either the password you entered was incorrect, or
```

Comment:
I couldn't get the test to pass for the full string. Something about the `'` character or word wrapping.